disperser/dataapi/metrics_handlers.go

```diff
@@ -17,19 +17,21 @@ const (
 )
 
 func (s *server) getMetric(ctx context.Context, startTime int64, endTime int64, limit int) (*Metric, error) {
-	operators, err := s.subgraphClient.QueryOperatorsWithLimit(ctx, limit)
```

Comment:
This is not only efficient, but also incorrect: if we have more than 10 operators (the default limit), we will not get the total stake.